doc/catalog-search-oai-harvest-process.md

```diff
@@ -0,0 +1,35 @@
+Catalog Search OAI Harvest Process
+===
+
+## Summary
+The current catalog OAI harvesting process is broken up into two distinct but linked processes and their respective DAGs:
+* [The Pre Production Catalog OAI Harvest](../cob_datapipeline/prod_catalog_oai_harvest_dag.py)
+* [The Production Catalog OAI Harvest](../cob_datapipeline/catalog_preproduction_oai_harvest_dag.py)
+
+Each of these DAGs will harvest independently to a configured collection:
+* `catalog_pre_production_oai_harvest`  harvests:
+    * from date set in `CATALOG_PRE_PRODUCTION_SOLR_COLLECTION` variable.
```

Comment:
Is this the right variable name?

Reply:
I believe so
![image](https://user-images.githubusercontent.com/444215/87187385-1a0db400-c2bb-11ea-8197-c427082ea20d.png)

Reply:
So I read this as stating that we are harvesting from a date that is extracted from the datestamp part of the collection name variable. I don't think that is correct.
Shouldn't it be that we are harvesting from  date set in `CATALOG_PRE_PRODUCTION_HARVEST_FROM_DATE`?

Reply:
Yeah, you're right. Copy pasta mistake.  Correcting.